specification/components/schemas/codings/ServiceRequest-StatusChanged.yaml

```diff
@@ -0,0 +1,13 @@
+title: representation of the date the service request state was set
+type: object
+required:
+  - code
+  - valueDateTime
+properties:
+  code:
+    type: string
+    enum:
+      - statusChanged
+  valueDateTime:
```

Comment:
I think we would want this as an [instant|https://hl7.org/fhir/R4/datatypes.html#instant] as in FHIR a dataTime can be a partial dateTime which I don't think we would want. @nhsd-jack-wainwright do you agree?